sp_mocks/wgp/backend/mock-database.js

```diff
@@ -23,12 +26,12 @@ module.exports = {
         {
             "email": "star",
             "password": "$2b$10$ssDWc4sXNoafqEdAsvH8TOUXywGsFHsPEODTZlSB4AKe8cqe1PmCi",
-            "profile_image": `${BACKEND_URL}/media/girl.png`,
+            "profile_image": `${BACKEND_URL}/anything/girl.png`,
```

Comment:
The code is all consistent here. However, when I pull the actual branch, the folder is not yet renamed. Am I missing something?

Reply:
Which folder are you talking about? 

Note: Here the thing is that `/media` is the route. which you will see in the Line 51 of BE: https://github.com/globe-and-citizen/layer8/blob/hmk/url-encrypt-decrypt-feat/sp_mocks/wgp/backend/server.js#L51 and here on Line 11 of FE: https://github.com/globe-and-citizen/layer8/blob/hmk/url-encrypt-decrypt-feat/sp_mocks/wgp/frontend/src/App.vue#L11. So what it does is that now that you have changed the routes only, not the folder name, so if I explain this in very simple words, if you just take a look on Line 51 of BE again: https://github.com/globe-and-citizen/layer8/blob/hmk/url-encrypt-decrypt-feat/sp_mocks/wgp/backend/server.js#L51 you will see `app.use("/anything", layer8_middleware.static("pictures"));` which is referrring to the folder named `pictures` so you can change your route to anything you want, it won't break the code.

Reply:
Got it. 
Thank you for the clarification.